src/tools/pub/cmd/pub-add.md

```diff
@@ -0,0 +1,70 @@
+---
+title: dart pub add
+description: Use dart pub add to add a dependency.
+toc: false
+---
+
+_Add_ is one of the commands of the [pub tool](/tools/pub/cmd).
+
+```
+$ dart pub add <package>[:<constraint>] [options]
+```
+
+This command adds the specified package to the pubspec as a dependency,
+and then gets the dependency.
+
+For example, the following command is equivalent to
+editing `pubspec.yaml` (adding `http: ^0.12.1` under `dependencies`)
+and then calling `dart pub get`:
```

Comment:
```suggestion
and then calling `dart pub get <package>`:
```

Reply:
I didn't know you could do this! (We don't document it in https://kw-www-dartlang-1.firebaseapp.com/tools/pub/cmd/pub-get, and I don't see it in the command-line help either.)

Reply:
my bad... you can't do `dart pub get <package>`, only `dart pub add <package>`.